zookeeper-examples/README.adoc

```diff
@@ -0,0 +1,19 @@
+# Vertx-Clustered-EventBus-with-Zookeeper
+
```

Comment:
Can add a few sentences explaining how to run the example. Typically how do I start the Zookeeper cluster, how do I run the example.

Reply:
there is a section in the end of the readme file explaining how to run on a single machine and on multiple machines, do you mean explaining how to run Zookeeper? If so I can add more details about how do I run Zookeeper on my machine

Reply:
Yes exactly. Typically I use Docker to run my Zookeeper server on my dev machine. I don't know what's the best: using docker or explaining how to install it and run it.